### PR TITLE
@damassi => auction2 fixes from QA session

### PR DESCRIPTION
**Pull request by sweir27**
Here are few fixes that came out of yesterday's QA session:
- Make the "works by artists you follow" carousel loop (so you can always click the right arrow)
works-loop

- Clicking the "Artists you Follow" checkbox when it is checked should _uncheck_ the box and show works by all artists
artists-you-follow

- Adds the currency symbol to the estimate slider
image

- Don't show "Your search returned 0 results" warning. I actually just changed this message to say "Loading", since in the current UI there's actually no way to get to 0 results, so that message was misleading. We can remove the message entirely if we want also /shrug.

### CODE REVIEW

File: desktop/components/main_layout/templates/index.jade [hunk 1]
```diff
@@ -5,8 +5,8 @@ block locals
 doctype html
 html(
   data-useragent=userAgent
-  data-user-type=sd.CURRENT_USER && sd.CURRENT_USER.type
-  data-lab-features=sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.join(',')
+  data-user-type=sd && sd.CURRENT_USER && sd.CURRENT_USER.type
```
Comment (sweir27) [inline desktop/components/main_layout/templates/index.jade]: So I received this error (the one we were seeing in prod where `sd` was not defined) locally... I'm not sure if this is the actual fix or just a bandaid.

Comment (sweir27) [inline desktop/components/main_layout/templates/index.jade]: I can remove it from this PR though if you think that's better.

File: desktop/apps/auction2/reducers.js [hunk 2]
```diff
@@ -124,12 +126,21 @@ function auctionArtworks(state = initialState, action) {
...
+            }
+          }, state)
+        } else {
+          return u({
+            filterParams: {
+              artist_ids: [],
+              include_artworks_by_followed_artists: true
+            }
+          }, state)
+        }
```
Comment (damassi) [inline desktop/apps/auction2/reducers.js]: So as to reduce duplication, you can probably collapse this into a one-line conditional that evaluates `state.filterParams.include_artworks_by_followed_artists`, then pass the result into a single updeep at the bottom.

**Comment by damassi** [comment]
👍

**Pull request merged**